tests/khmer_tst_utils.py

```diff
@@ -64,17 +84,16 @@ def _runscript(scriptname, sandbox=False):
         return 0
     except pkg_resources.ResolutionError as err:
         if sandbox:
-            paths = [os.path.join(os.path.dirname(__file__), "../sandbox")]
+            path = os.path.join(os.path.dirname(__file__), "../sandbox")
         else:
-            paths = [os.path.join(os.path.dirname(__file__),
-                                  "../scripts")]
-            paths.extend(os.environ['PATH'].split(':'))
-        for path in paths:
-            scriptfile = os.path.join(path, scriptname)
+            path = scriptpath()
```

Comment:
On Mon, Jul 27, 2015 at 10:51:16AM -0700, Michael R. Crusoe wrote:
> >          else:
> > -            paths = [os.path.join(os.path.dirname(__file__),
> > -                                  "../scripts")]
> > -            paths.extend(os.environ['PATH'].split(':'))
> > -        for path in paths:
> > -            scriptfile = os.path.join(path, scriptname)
> > +            path = scriptpath()
> 
> should be `path = scriptpath(scriptname)`

*shrug* we're using scriptpath() elsewhere for cleanliness reasons. I actually
think we should remove 'scriptname' from the argument list and just hardcode
it, but I left it in just in case.